templates/types/streaming/fastapi/app/tasks/llama_cloud.py

```diff
@@ -0,0 +1,89 @@
+import os
+import logging
+import requests
+from typing import List, Any, Dict, Optional
+
+
+logger = logging.getLogger("uvicorn")
+
+
+class LLamaCloudFile:
```

Comment:
only two methods are public, can we mark the private methods, so it's clear?
(same for the other file service)